08_project/alextmz/pkg/puppy/store/mapstore.go

```diff
@@ -0,0 +1,50 @@
+package store
+
+import (
+	pp "github.com/anz-bank/go-course/08_project/alextmz/pkg/puppy"
```

Comment:
in go world people are using long names like `puppy` for accessing modules and short ones like a `p` for vars

Reply:
Agree, but it becomes a cacophony of puppy.Puppy... changed it as asked because I couldn't find a shorter name that made more sense :)